windows/src/developer/kmcomp/Keyman.Developer.System.ValidateRepoChanges.pas

```diff
@@ -0,0 +1,463 @@
+unit Keyman.Developer.System.ValidateRepoChanges;
```

Comment:
This unit is really for internal use only -- I included it in kmcomp at present for testing and we may remove it again later. I don't want to spend effort documenting it at present :)